### PR TITLE
Use React framework to rewrite Mars UI

### PR DESCRIPTION
**Pull request by wjsi**
## What do these changes do?

Rewrite Mars web pages using React framework and Material UI.

### CODE REVIEW

File: mars/services/scheduling/worker/quota.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ class QuotaActor(mo.Actor):
     def gen_uid(cls, band):
         return f'{band}_quota'
 
-    def __init__(self, band, quota_size, manager_ref=None):
+    def __init__(self, band, quota_size):
```
Comment (qinxuye) [inline mars/services/scheduling/worker/quota.py:45]: Lack of type annotation.

**Review by qinxuye — COMMENTED** [review]
LGTM overall

**Review by qinxuye — APPROVED** [review]
LGTM

**Review by hekaisheng — APPROVED** [review]
LGTM

**Pull request merged**